src/main/webapp/WEB-INF/jsp/webpay_plus/aborted.jsp

```diff
@@ -0,0 +1,39 @@
+<head>
+    <meta charset="UTF-8">
+
+</head>
+<jsp:include page="../template/tpl-header.jsp" />
+
+<body class="container">
+    <jsp:include page="../template/navbar-webpayplus.jsp" />
+
+<style>
+	.row {
+		padding-bottom: 20px;
+	}
+</style>
+
+<div class="row">
+	<div class="col">
+		 <h2>Compra cancelada por el usuario</h2>
+         <p>Luego de que se anula la compra en el formulario de pago recibir�s un POST con lo siguiente:</p>
```

Comment:
Are we sure that it's a POST and not a GET request?

Reply:
The corresponding will be made